src/api/catalog.js

```diff
@@ -68,21 +68,21 @@ export default ({config, db}) => function (req, res, body) {
 		url = config.elasticsearch.protocol + '://' + url
 	}
 
-	// Check price tiers
```

Comment:
@ResuBaka why checking price tiers part is gone?

Reply:
I have removed it because I need the user group now too. So it was easier to remove the if.

I does not destroy the logic for PriceTiers as it is checked before the actual logic for it is executed anyways.  

Which can be seen [here](https://github.com/DivanteLtd/vue-storefront-api/pull/307/files/d946f4f76355a1d1d9d9bf526e089ba2f7c13eca#diff-1634c1caa0a81468b05730cb1764f208L59) and [here ](https://github.com/DivanteLtd/vue-storefront-api/pull/307/files/d946f4f76355a1d1d9d9bf526e089ba2f7c13eca#diff-f53f0b2d2a49e9ec0d701c7d41a3451aR60).